src/prng/hc128.rs

```diff
@@ -514,20 +517,21 @@ mod test {
                  0xad, 0x83, 0x6b, 0x9d, 0x60, 0xa1, 0x99, 0x96,
                  0x90, 0x00, 0x66, 0x7f, 0xfa, 0x7e, 0x65, 0xe9,
                  0xac, 0x8b, 0x92, 0x34, 0x77, 0xb4, 0x23, 0xd0,
-                 0xb9, 0xab, 0xb1, 0x47, 0x7d, 0x4a, 0x13, 0x0a);
+                 0xb9, 0xab, 0xb1, 0x47, 0x7d, 0x4a, 0x13, 0x0a];
 
         // Pick a somewhat large buffer so we can test filling with the
         // remainder from `state.results`, directly filling the buffer, and
         // filling the remainder of the buffer.
-        let mut buffer = vec!(0u8; 16*4*2);
+        let mut buffer = [0u8; 16*4*2];
         // Consume a value the we have a remainder.
         let _ = rng.next_u64();
         rng.fill_bytes(&mut buffer);
 
-        for i in buffer.iter() {
-            print!("0x{:02x}, ", i);
+        // [u8; 128] doesn't miplement PartialEq
```

Comment:
Small typo. But I shouldn't say anything, because in the comment above it there is a typo by me :smile: